src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -168,6 +170,34 @@ public static Set<Cca> parseCcas(Collection<String> ccas) throws ParseException
     }
 
     /**
+     * Parses a {@code String attendance} into an {@code attendance}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code attendance} is invalid.
+     */
+    public static Attendance parseAtt(String attendance) throws ParseException {
+        requireNonNull(attendance);
+        String trimmedAtt = attendance.trim();
+        if (!Attendance.isValidAttendance(trimmedAtt)) {
+            throw new ParseException(Attendance.MESSAGE_CONSTRAINTS);
+        }
+        return new Attendance(attendance);
+    }
+
+    /**
+     * Parses a {@code String sessions} into an {@code sessions}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code sessions} is invalid.
+     */
+    public static Sessions parseSess(String sessions) throws ParseException {
+        requireNonNull(sessions);
+        String trimmedSess = sessions.trim();
+        if (!Sessions.isValidSessions(trimmedSess)) {
+            throw new ParseException(Sessions.MESSAGE_CONSTRAINTS);
+        }
+        return new Sessions(sessions);
+
      * Parses {@code Metadata metadata} into a {@code Metadata}.
```

Comment:
I think you should comment this correctly - I think its causing error to the source code